src/Settings.php

```diff
@@ -158,6 +164,11 @@ public static function parseArguments(array $arguments)
                         $settings->parallelJobs = max((int) $arguments->getNext(), 1);
                         break;
 
+                    case '--colors':
+                        $settings->forceColors = true;
+                        $settings->colors = true;
```

Comment:
I think better solution is to change `colors` variable from boolean to somethink like Enum with three possible values `DISABLED`, `AUTODETECT` and `FORCE` and then remove `forceColors` variable. I think then was clear, what every variant do.